src/coreboot.u-root.systemboot/index.md

```diff
@@ -1,59 +1,84 @@
 # LinuxBoot using coreboot, u-root and systemboot
 
-Points of contact: [Andrea Barberio](https://github.com/insomniacslk), [David Hendricks](https://github.com/dhendrix)
+Points of contact:
+[Andrea Barberio](https://github.com/insomniacslk),
+[David Hendricks](https://github.com/dhendrix)
 
-This chapter describes how to build a LinuxBoot firmware based on coreboot, u-root and systemboot.
-The examples will focus on `x86_64`, and the coreboot builds will cover virtual and physical OCP hardware.
+This chapter describes how to build a LinuxBoot firmware based on coreboot,
+u-root and systemboot.  The examples will focus on `x86_64`, and the coreboot
+builds will cover virtual and physical OCP hardware.
 
 ## Quick Start with coreboot
 
-Run these commands in a directory you create or in /tmp; do so because it creates some files and directories:
+Run these commands in a directory you create or in `/tmp`; do so because it
+creates some files and directories:
 
-	$ go get github.com/linuxboot/corebootnerf
-	$ go run github.com/linuxboot/corebootnerf --fetch
-	... lots and lots of output!
+```
+$ go get github.com/linuxboot/corebootnerf
```

Comment:
just spotting this here (not PR related), we gotta rework such things since Go changed in terms of get/install semantics